CHANGELOG.rst

```diff
@@ -3,6 +3,16 @@ Changelog
 =========
 
 
+2.0.3 (unreleased)
+==================
+* Prevent changes to ``DJANGOCMS_LINK_XXX`` settings from requiring new
+  migrations
+* Changed naming of ``Aldryn`` to ``Divio Cloud``
+* Adapted testing infrastructure (tox/travis) to incorporate
+  django CMS 3.4 and dropped 3.2
+* Removed NetBios Pattern from documentations as its faulty
```

Comment:
correct one would have been ```DJANGOCMS_LINK_INTRANET_HOSTNAME_PATTERN = r"[a-z,0-9,!@#$%^()\\\-'{}.~]{1,15}"``` I discussed with Stefan, should I still add this info? It was only once used for the intranet.